config.json5

```diff
@@ -0,0 +1,31 @@
+{
+    // Express app configs
+    server: {
+        // The port that the personal data server will listen to.
+        // Can be overridden by the value defined in the environment variable SERVERPORT
+        port: 3000
+    },
+
+    // Database configs
+    db: {
+        // The docker container name for personal server postgres db
+        dbContainerName: "PersonalDataPostgres",
+        // The postgres docker image version
+        dbDockerImage: "postgres:14.0-alpine",
```

Comment:
This doesn't seem to be one of the "supported tags" as per https://hub.docker.com/_/postgres - is it that we have a choice of 14.1-alpine or 14-alpine?